src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -755,6 +755,37 @@ GUI_TEST_CLASS_DEFINITION(test_7384_2) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7401) {
+    // 1. Open human_T1.fa.
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+
+    // 2. Select any part of sequence.
+    auto panView = GTUtilsSequenceView::getPanViewByNumber(os);
+    CHECK_SET_ERR(panView != nullptr, "No pan view found!");
+
+    auto startPoint = panView->mapToGlobal(panView->rect().center());
+    auto endPoint = QPoint(startPoint.x() + 150, startPoint.y());
+    GTMouseDriver::moveTo(startPoint);
+    GTMouseDriver::press();
+    GTMouseDriver::moveTo(endPoint);
+    GTMouseDriver::release();
+
+    // 3. Move mouse a bit upper
+    endPoint = QPoint(endPoint.x(), endPoint.y() - 20);
+    GTMouseDriver::moveTo(endPoint);
+
+    // 4. Double click and move the cursor to the right (or to the left).
+    GTMouseDriver::click();
```

Comment:
this is a click, not a double click. The 'click' is designed to be safe for double click detection: click + click != double click in general in GT API

Reply:
Actually, it's pretty the same:
![image](https://user-images.githubusercontent.com/26257527/134160112-77f65a55-91c3-4eaf-8dfa-ae84af4a3ce4.png)

I could add `sleep` if you want

Reply:
The idea is that I cannot use `GTMouseDriver::doubleClick` because I need to move mouse before the last `release` is processed

Reply:
Ok lets keep it "special" and check that both Linux & Windows tests behave as expected

Reply:
OK